python/ml4ir/base/model/architectures/dnn.py

```diff
@@ -21,7 +22,22 @@ class DNNLayer:
 class DNN:
     def __init__(self, model_config: dict, feature_config: FeatureConfig, file_io):
         self.file_io: FileIO = file_io
+        self.model_config = model_config
+        self.feature_config = feature_config
         self.layer_ops: List = self.define_architecture(model_config, feature_config)
+        if 'positional_bias_handler' in self.model_config and self.model_config['positional_bias_handler'][
+            'key'] == PositionalBiasHandler.MLRANKER:
+            self.positional_bias_layer = layers.Dense(1,
+                                            name="positional_bias_layer",
+                                            activation=None,
+                                            use_bias=False,
+                                            kernel_initializer="glorot_uniform",
+                                            bias_initializer="zeros",
+                                            kernel_regularizer=None,
+                                            bias_regularizer=None,
+                                            activity_regularizer=None,
+                                            kernel_constraint=None,
+                                            bias_constraint=None)
```

Comment:
@mohazahran These are defaults anyway, right? Why fill them out explicitly?

Reply:
These were for sanity during development.